ProductVariantDraft#getPrices() could be null  despite it is not marked as @Nullable

`ProductVariantDraft#getPrices()` list is not marked as `@Nullable` (like, for example `ProductVariantDraft#getAssets()`), thus it's not expected to have it **null** in actual draft instances.

But `ProductVariantDraftBuilderBase#of()` create instance with **null** prices: 
```java
ProductVariantDraftDsl draft = ProductVariantDraftBuilder.of().build();
assertThat(draft.getPrices()).isNotNull();
```
> java.lang.AssertionError: 
> Expecting actual not to be null

This is documentation or implementation issue: either mark it `@Nullable` as `getAssets()` or implement it so it is never **null**

Also, in `ProductVariantDraft` the same issue with `getAttributes` and `getImages`.
Guess other entities drafts could have the same issue

--- a/commercetools-internal-docs/src/main/java/io/sphere/sdk/meta/ReleaseNotes.java
+++ b/commercetools-internal-docs/src/main/java/io/sphere/sdk/meta/ReleaseNotes.java
@@ -161,6 +161,7 @@ import java.util.function.Function;
     <li class=change-in-release> {@link PriceDraft#getChannel()} is of type {@link ResourceIdentifier<Channel>} instead of {@link Reference<Channel>}</li>
     <li class=change-in-release> removed @Deprecated annotation from {@link io.sphere.sdk.carts.commands.updateactions.AddLineItem#of(String, int, long)}</li>
     <li class=change-in-release> removed @Deprecated annotation from {@link io.sphere.sdk.carts.commands.updateactions.AddLineItem#of(ProductIdentifiable, int, long)}</li>
+    <li class=change-in-release> {@link ProductVariantDraft#getPrices()}, {@link ProductVariantDraft#getAttributes} and {@link ProductVariantDraft#getImages()} are now optional</li>
  </ul>
 
  <h3 class=released-version id="v1_38_0">1.38.0 (07.12.2018)</h3>

--- a/commercetools-models/src/main/java/io/sphere/sdk/products/ProductVariantDraft.java
+++ b/commercetools-models/src/main/java/io/sphere/sdk/products/ProductVariantDraft.java
@@ -22,10 +22,13 @@ public interface ProductVariantDraft {
     @Nullable
     String getSku();
 
+    @Nullable
     List<PriceDraft> getPrices();
 
+    @Nullable
     List<AttributeDraft> getAttributes();
 
+    @Nullable
     List<Image> getImages();
 
     @Nullable

--- a/commercetools-models/src/main/java/io/sphere/sdk/products/ProductVariantDraftBuilder.java
+++ b/commercetools-models/src/main/java/io/sphere/sdk/products/ProductVariantDraftBuilder.java
@@ -13,7 +13,7 @@ import static java.util.Arrays.asList;
 
 public final class ProductVariantDraftBuilder extends ProductVariantDraftBuilderBase<ProductVariantDraftBuilder> {
 
-    ProductVariantDraftBuilder(@Nullable final List<AssetDraft> assets, final List<AttributeDraft> attributes, final List<Image> images, @Nullable final String key, final List<PriceDraft> prices, @Nullable final String sku) {
+    ProductVariantDraftBuilder(@Nullable final List<AssetDraft> assets, @Nullable final List<AttributeDraft> attributes, @Nullable final List<Image> images, @Nullable final String key, @Nullable final List<PriceDraft> prices, @Nullable final String sku) {
         super(assets, attributes, images, key, prices, sku);
     }
 

